2 table.sql

```diff
@@ -0,0 +1,29 @@
+use BicycleShopDb
+go
+drop table if exists dbo.Shop
+go
+create table dbo.Shop(
+    BikeId int not null identity primary key,
+    FirstName varchar(20) not null constraint c_Shop_first_name_cannot_be_blank check(FirstName > ''),
+    LastName varchar(20) not null constraint c_Shop_last_name_cannot_be_blank check(LastName > ''),
+    StreetAddress varchar(50) not null constraint c_Shop_street_address_cannot_be_blank check(StreetAddress > ''),
+    PhoneNumber varchar(15) not null constraint c_Shop_phone_number_must_contain_digits_only check(isnumeric(PhoneNumber) = 1),
+    BikeCompany varchar(30) not null constraint c_Shop_bike_company_cannot_be_blank check(BikeCompany > ''),
+    BikeSize varchar(2) not null constraint c_Shop_bike_size_must_be_one_or_two_digits check(BikeSize like '[1-9]' or BikeSize like '[0-9][0-9]'),
+    BikeColor varchar(15) not null constraint c_Shop_bike_color_cannot_be_blank check(BikeColor > ''),
+    DatePurchased date not null constraint c_Shop_date_purchased_must_be_on_or_after_March_1_2022 check(DatePurchased >= '03-01-2022'),
+    PurchasePrice decimal(6,2) not null,
+    NewOrUsed varchar(4) not null constraint c_Shop_new_or_used_only_allows_new_or_used check(NewOrUsed in ('new', 'used')),
+    ConditionWhenReceived varchar(11) null constraint c_Shop_condition_when_received_only_allows_perfect_minor_or_major_fixup_or_restoration
```

Comment:
I think it would make more sense to save this as a number from one to 4. It would make it easier to save records. At this point you don't have to change it, as changing it will affect the other files.

Reply:
Could you please clarify why recording as a number makes it easier to save the records?

Reply:
The way you are doing it you would have to type out the name each time you enter it. Recording it as a int will also save memory. Its no big difference in this case.